cdap-docs/developers-manual/source/building-blocks/workflows.rst

```diff
@@ -181,9 +181,9 @@ The workflow itself uses the same names in its configuration::
 
 Concurrent Workflows
 --------------------
-By default, a workflow runs sequentially. Multiple instances of a workflow can be run
-concurrently. To enable concurrent runs for a workflow, set its runtime argument
-``concurrent.runs.enabled`` to ``true``.
+By default, a workflow runs concurrently, allowing multiple instances of a workflow to be run
+simultaneously. To disable concurrent runs for a workflow, set its runtime argument
+``concurrent.runs.enabled`` to ``false``.
```

Comment:
@sagarkapare this was removed wasn't it? As far as I know, concurrency is a property of schedules (http://docs.cdap.io/cdap/current/en/developers-manual/building-blocks/schedules.html#run-constraints) and not of the workflow itself.